parachain/pallets/control/src/lib.rs

```diff
@@ -122,6 +122,13 @@ where
 	No,
 }
 
+pub struct AllowSiblingsOnly;
```

Comment:
I'd prefer that this goes into our core primitives crate, as its very general.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/989/commits/e3e6c8d915c8451c3c583eb5ae8396e246f23d43